api/test/unit/controllers/test_search_controller_search_query.py

```diff
@@ -129,11 +129,11 @@ def test_create_search_query_q_search_with_quotes_adds_raw_suffix(
         ],
         "should": [
             {
-                "simple_query_string": {
-                    "boost": 10000,
-                    "fields": ["title"],
-                    "query": "The cutest cat",
-                    "flags": DEFAULT_SQS_FLAGS,
+                "match_phrase": {
+                    "title": {
+                        "query": '"The cutest cat"',
```

Comment:
For other reviewers who notice this double-quoting. It's preserved from the input string. It shouldn't be a problem, it seems that (at least with our current analyser configuration for title) `match_phrase` filters our non-alphanumeric tokens in search. For example, a query for `"house cat photo"` (with the quotes) ends up as this, for the `match_phrase` part of the scoring, and is identical if I drop the quotes in the query string.

```json
{
  "value": 10000,
  "description": "weight(title:\"hous cat photo\" in 95) [PerFieldSimilarity], result of:",
  "details": [
    {
      "value": 10000,
      "description": "score(BooleanWeight), computed from:",
      "details": [
        {
          "value": 10000,
          "description": "boost, query boost",
          "details": []
        }
      ]
    }
  ]
}
```